### PR TITLE
Alert and buttons

### LINKED ISSUE #1: Button Component
### Buttons
- [x] Basic Buttons
- [x] Disabled Buttons
- [x] Outline Buttons
- [x] Link Buttons
- [x] Icon Buttons
- [x] Floating Action Button

### LINKED ISSUE #3: Alerts
- [x] Basic alerts
- [x] Alerts with icons
- [x] Alerts with dismiss

### PR DESCRIPTION
**Pull request by siddhigate**
Alert component
- Basic alert
- Alert with icons
- Alert with dismiss

Buttons component
- Basic buttons
- Outline buttons
- Buttons with icons
- Link 
- Disabled buttons
- Sized buttons
- Floating action buttons

**Preview:** https://upsleek-css-component-library-git-alert-and-buttons-siddhigate.vercel.app/

### CODE REVIEW

**Comment by pritam-kr** [comment]
Everythings looks good

**Review by pritam-kr — APPROVED** [review]

File: css/components/alert.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
+.alert {
+  border-radius: 0.3rem;
```
Comment (BelideNikhil) [inline css/components/alert.css:2]: Don't use REM for border-radius as they'll become shape-shifters based on pixel density.
Rest looks good!

**Review by BelideNikhil — APPROVED** [review]
image

flex is interrupting here, all the buttons visually look the same even though they are actually different

File: css/components/alert.css (hunk 1 above)
Comment (siddhigate) [reply css/components/alert.css:2]: Done!

**Pull request merged**